modules/ROOT/pages/clustering/settings.adoc

```diff
@@ -90,6 +90,8 @@ Possible values are:
 
 * V2_ONLY -- it runs only discovery service v2.
```

Comment:
This is a good idea. I'll think how to fix it

Reply:
@nick-giles-neo hi! Could you please take a look at this page one more time? I've updated the table

Reply:
Commented out of this thread. I think the change is better, because it acknowledges the V2 settings, but I think it needs to lean more on 'if you are using V2 then...' rather than the product version being 5.23.